src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header-img {
+  display: inline-block;
+  margin: 10px 0 10px 50px;
+}
+
+.header-nav {
+  margin-right: 50px;
+}
+
+.nav-list {
+  display: flex;
+  padding: 0;
+  margin-top: 20px;
+}
+
+.nav-item {
+  position: relative;
+  white-space: nowrap;
+  list-style: none;
+}
+
+.nav-item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav-link {
+  font-family: Roboto, sans-serif;
```

Comment:
move it to `html`
add one widespread sans-serif font: `font-family: Roboto, Arial, sans-serif;`

Reply:
did so, but now the tests fail

Reply:
a bit weird, but the main thing is that works properly (font-family applied): 
![image](https://user-images.githubusercontent.com/86923089/191079178-73b59fd3-0e30-4ef3-93b8-a448cbaeb2bb.png)

of course, it's important to have tests passed but taking into account the above - approved :+1: 
P.S. Just wanted you to know that we should define font-family on the highest level